### PR TITLE
feat: add `WAIT` command

### PR DESCRIPTION
**Pull request by kwannoel**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

Closes https://github.com/risingwavelabs/risingwave/issues/12885

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

File: src/meta/src/rpc/ddl_controller.rs [hunk 1]
```diff
@@ -1094,4 +1096,15 @@ impl DdlController {
...
+
+    pub async fn wait(&self) {
+        while !self
+            .catalog_manager
+            .list_creating_background_mvs()
+            .await
+            .is_empty()
+        {
+            sleep(Duration::from_secs(1)).await;
+        }
```
Comment (kwannoel) [inline src/meta/src/rpc/ddl_controller.rs:1108]: The wait logic.

**Review by chenzl25 — APPROVED** [review]
Generally LGTM. Should we support canceling `wait` statement or set a maximum wait time to avoid a potential dead loop?

**Comment by BugenZhao** [comment]
Will this command be user-facing?

**Comment by BugenZhao** [comment]
> avoid a potential dead loop

It's not a big deal. The loop will be cancelled once the gRPC connection closes, that is, the frontend cancels the request.

**Comment by kwannoel** [comment]
> Will this command be user-facing?

Nope. Just for tests.

**Pull request merged**